test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -35,19 +39,19 @@ def build_transfer_table() -> [[int]]:
 # create an erc20 contract for rowan via lock rowan in sifnode
 def burn_rowan_get_erc20_address(ctx: test_utils.EnvCtx) -> Address:
     # check contract not created before
-    token_address = ctx.get_destination_contract_address(rowan)
+    token_address = ctx.get_destination_contract_address("rowan")
     if token_address != NULL_ADDRESS:
         return token_address
     fund_amount_sif = 10 ** 20
     fund_amount_eth = 10 ** 20
     rowan_transfer_amount = 10 ** 18
-    test_sif_account = ctx.create_sifchain_addr(fund_amounts={rowan:fund_amount_sif, ctx.ceth_symbol:fund_amount_eth})
```

Comment:
Please use `dict` objects for balances, like this:
```
{"denom1": 100, "denom2": 200}
```

You will find all the functions for arithmetic operations on balances in `siftool.cosmos` package.

The old way of using lists (such as `[[100, "denom1"], [200, "denom2"]]` is deprecated since it was problematic for several reasons. It will be eventually cleaned up.

Reply:
will use map to replace array.